packages/engine/src/scene/functions/createGround.ts

```diff
@@ -33,5 +37,15 @@ export const createGround = async function (entity: Entity, args: GroundProps, i
 
   createCollider(entity, mesh)
 
+  if (isClient) {
```

Comment:
lets move this to the navigation system and add a GroundPlaneTagComponent component to query for, to keep this decouple from the core engine functionality

Reply:
For now, maybe we can just have an option in the editor on the Scene node for if the scene should have navmeshes or not. That way we can optimise scenes that don't need it, and have a single source of truth per-secne.